src/applications/vaos/new-appointment/components/VAFacilityPage/VAFacilityPageV2.jsx

```diff
@@ -0,0 +1,101 @@
+import React, { useEffect } from 'react';
+import { connect } from 'react-redux';
+import { scrollAndFocus } from '../../../utils/scrollAndFocus';
+import LoadingIndicator from '@department-of-veterans-affairs/formation-react/LoadingIndicator';
+import SchemaForm from 'platform/forms-system/src/js/components/SchemaForm';
+
+import * as actions from '../../redux/actions';
+import { getFacilityPageV2Info } from '../../../utils/selectors';
+import { FETCH_STATUS } from '../../../utils/constants';
+import FacilitiesRadioWidget from './FacilitiesRadioWidget';
+import FormButtons from '../../../components/FormButtons';
+
+const initialSchema = {
+  type: 'object',
+  required: ['vaFacility'],
+  properties: {
+    vaFacility: {
+      type: 'string',
+      enum: [],
+    },
+  },
+};
+
+const uiSchema = {
+  vaFacility: {
+    'ui:title': 'Please select where you’d like to have your appointment.',
+    'ui:widget': FacilitiesRadioWidget,
+  },
+};
+
+const pageKey = 'vaFacilityV2';
+const pageTitle = 'Choose a VA location for your appointment';
+
+function VAFacilityPageV2({
+  schema,
+  data,
+  facilities,
+  childFacilitiesStatus,
+  openFacilityPageV2,
+  updateFacilityPageV2Data,
+  typeOfCare,
+  routeToPreviousAppointmentPage,
+  pageChangeInProgress,
+}) {
```

Comment:
This custom radio widget needed to be created for this page for 2 reasons:
1. The dynamic content of our enumOptions and the need to display custom JSX for each label
1. The expand link